<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>replaceDialog</class>
 <widget class="QDialog" name="replaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="findButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>find next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replaceButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>replace</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replaceAllButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>replace all</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="findEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>41</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="replaceEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>101</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>change</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>63</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>to</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
